IdP/storage.go

```diff
@@ -60,13 +61,49 @@ func (s *storage) GetUser(ctx context.Context, userID string) (User, error) {
 // ChangePassword allows a user to change their password.
 func (s *storage) ChangePassword(ctx context.Context, userID string, password string) error {
 	// TODO(bimmlerd)
-	return fmt.Errorf("not implemented")
+	//hash password
+	h := sha1.New()
+	pwHash := h.Sum([]byte(password))
+
+	//does the user exist?
+	u := dbUser{}
```

Comment:
I don't think you need to scan all of the dbUser, I think just `SELECT`ing the `uid` should be enough